server/src/resolvers/Attribute.ts

```diff
@@ -132,3 +132,36 @@ export const deleteAttributes: FieldResolver<
   })
   return res
 }
+
+export const createInputGroup: FieldResolver<
+  'Mutation',
+  'createInputGroup'
+> = async (_parent, { attributeId }, ctx) =>
+  ctx.prisma.attribute.update({
```

Comment:
would be clearer to use prisma.inputGroups.create({attribute: {connect: attributeId}})

Reply:
I wanted to have a mutation on Attribute to make it easier to use the cache in the client.
Not sure if it would be as easy with your solution, I'll have to try

Reply:
ok I get it, it can be confusing that a mutation "createInputGroup" returns an attribute (you have to really understand the data model) but since it will soon disappear I won't insist lol